src/views/certifications/views/[slug]/components/program-hero/program-hero.module.css

```diff
@@ -0,0 +1,26 @@
+.heroBackground {
+	background: var(--background-image);
+	background-position: top center;
+	background-repeat: no-repeat;
+	background-size: cover;
+	bottom: -10%;
+	filter: blur(8px);
+	left: -10%;
+	overflow: hidden;
+	position: absolute;
+	right: -10%;
+	top: -10%;
+	z-index: -1;
+
+	&.theme-infrastructure-automation {
+		--background-image: url('./assets/infrastructure-automation.jpg');
```

Comment:
<img width="1130" alt="Screen Shot 2022-12-13 at 2 37 35 PM" src="https://user-images.githubusercontent.com/36613477/207460004-bebc05d3-b16f-41ac-b736-513ca16cebf9.png">

Maybe these are recent changes but it looks like the backgrounds are updated with patterns on top of the gradient?

Reply:
Great eye 👀 These are recent changes! I'm planning on updating the assets once everything is finalized in the design (I don't think it was yet when I opened this PR, I think it might have just been finalized this morning!)